text/chapter5.md

```diff
@@ -373,8 +380,6 @@ For example, the `Line` constructor defined above required two `Point`s, so to c
 {{#include ../exercises/chapter5/src/Data/Picture.purs:exampleLine}}
 ```
 
-To construct the points `p1` and `p2`, we apply the `Point` constructor to its single argument, which is a record.
```

Comment:
(`Point` must have been changed at some point from a newtype to just a plain record)

Reply:
Indeed. In https://github.com/purescript-contrib/purescript-book/pull/254. Good catch.